Remotion/Data/DomainObjects/Queries/QueryFactory.cs

```diff
@@ -279,7 +279,7 @@ public static IQuery CreateCollectionQuery (
       ArgumentNullException.ThrowIfNull(queryParameterCollection);
       ArgumentNullException.ThrowIfNull(collectionType);
 
-      var definition = new QueryDefinition(id, storageProviderDefinition, statement, QueryType.CollectionReadOnly, collectionType, metaData);
+      var definition = new QueryDefinition(id, storageProviderDefinition, statement, QueryType.CollectionReadOnly, QueryStatementType.Text, collectionType, metaData);
```

Comment:
since the other occuranes aren't changed too, we should either leave it as a default here as well or change all instantiations of QueryDefinition in the factory.